web/src/components/FormApdOverview.js

```diff
@@ -5,27 +5,38 @@ import { Field, reduxForm } from 'redux-form';
 import { Textarea } from './Inputs';
 import SectionHeader from './SectionHeader';
 
-const FormApdOverview = ({ handleSubmit }) => (
+const FormApdOverview = ({ handleSubmit, submitting }) => (
   <form onSubmit={handleSubmit}>
     <SectionHeader>
       Briefly describe the high-level vision of your program. What are you
       trying to accomplish by 2020?
     </SectionHeader>
-    <Field name="vision" component={Textarea} label="Vision" />
+    <Field name="program_vision" component={Textarea} label="Vision" />
     <SectionHeader>
       In 1–2 sentences, summarize the key benefits of the program:
     </SectionHeader>
-    <Field name="benefits" component={Textarea} label="Benefits" />
+    <Field name="program_benefits" component={Textarea} label="Benefits" />
+    <div className="mt3">
+      <button
+        type="submit"
+        className="btn btn-primary bg-green"
+        disabled={submitting}
+      >
+        {submitting ? 'Saving' : 'Submit'}
+      </button>
+    </div>
   </form>
 );
 
 FormApdOverview.propTypes = {
-  handleSubmit: PropTypes.func.isRequired
+  handleSubmit: PropTypes.func.isRequired,
+  submitting: PropTypes.bool.isRequired
 };
 
 const formConfig = {
   form: 'apdOverview',
-  destroyOnUnmount: false
+  destroyOnUnmount: false,
```

Comment:
I wonder if we can get rid of the `destroyOnUnmount` for the forms that are wired up now.  We take the submitted data, send it to the API, then update the app state with the returned data.  That is then used to set the `initialValues`, so if the form is destroyed when we move on to the next section, it's no big deal - it gets rebuilt with the data the API sent back after saving previously.  Is that right?  I'm not sure if it's a big deal, but I assume there's a reason it defaults to destroying the forms. 🤷‍♂️